Pipfile

```diff
@@ -13,4 +13,4 @@ pytest = "*"
 review-wof = {editable = true,path = "."}
 
 [requires]
-python_version = "3.6"
+python_version = "3.7"
```

Comment:
I'd like to leave the current version as-is, 3.7 is not conveniently available on RHEL7, while 3.6 is in EPEL.

Reply:
I am working on my home laptop which is MAC. Can we find a middle ground which works for both of us?